cargo/flatpak-cargo-generator.py

```diff
@@ -146,6 +153,9 @@ async def get_dep_packages(entry, toml_dir):
                 if dep_name in packages:
                     continue
                 dep_dir = os.path.normpath(os.path.join(toml_dir, dep['path']))
+                if is_excluded(dep_dir):
```

Comment:
I believe that an excluded member of a workspace is just excluded from the list of members of the workspace. If a package is listed as a dependency, it will be needed during compilation so it must be resolved somehow so it doesn't make sense to not try to load it.